lasagne/layers/special.py

```diff
@@ -530,3 +534,102 @@ def prelu(layer, **kwargs):
     if nonlinearity is not None:
         layer.nonlinearity = nonlinearities.identity
     return ParametricRectifierLayer(layer, **kwargs)
+
+
+class RandomizedRectifierLayer(Layer):
+    """
+    lasagne.layers.RandomizedRectifierLayer(incoming,
+    lower=init.Constant(0.3), upper=init.Constant(0.8),
+    shared_axes='auto', **kwargs)
+
+    The randomized leaky rectifier was first proposed and used in the Kaggle
```

Comment:
Ah, correct... it's not necessarily the full first sentence, but everything up to the first dot or line break. The ParametricRectifierLayer was lucky,